src/apps/replication/meta_server/meta_service.cpp

```diff
@@ -118,6 +112,12 @@ error_code meta_service::start()
         return err;
     }
 
+    _balancer = new load_balancer(_state);
+    // make sure the delay is larger than fd.grace to ensure
+    // all machines are in the correct state (assuming connected initially)
+    tasking::enqueue(LPC_LBM_START, this, &meta_service::on_load_balance_start, 0,
+        _opts.fd_grace_seconds * 1000);
+
     // register rpc handlers
     register_rpc_handler(
         RPC_CM_QUERY_NODE_PARTITIONS,
```

Comment:
Yes, I didn't realize the problem. I already fixed it in https://github.com/qinzuoyan/rDSN/commit/98047589679d7fbac5b4bb0aa043bf6d1896ef43 .